ibm/resource_ibm_cdn.go

```diff
@@ -0,0 +1,579 @@
+package ibm
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/softlayer/softlayer-go/sl"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/softlayer/softlayer-go/datatypes"
+	"github.com/softlayer/softlayer-go/services"
+)
+
+const str string = ".cdnedge.bluemix.net"
+
+func resourceIBMCDN() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceIBMCDNCreate,
+		Read:   resourceIBMCDNRead,
+		Update: resourceIBMCDNUpdate,
+		Delete: resourceIBMCDNDelete,
+		Exists: resourceIBMCDNExists,
+
+		Schema: map[string]*schema.Schema{
+			"hostname": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"vendor_name": &schema.Schema{
```

Comment:
https://api.softlayer.com/rest/v3/SoftLayer_Network_CdnMarketplace_Vendor/listVendors.json...Since we have one vendor ...We can make vendor as optional and set the default value to "akamai"